internal/controller/controllers/preprovisioningimage_controller.go

```diff
@@ -319,6 +320,17 @@ func (r *PreprovisioningImageReconciler) AddIronicAgentToInfraEnv(ctx context.Co
 }
 
 func (r *PreprovisioningImageReconciler) getIronicAgentImage(log logrus.FieldLogger, infraEnv common.InfraEnv) (string, error) {
+	version := semver.New(infraEnv.OpenshiftVersion)
+	// ignore PreRelease, anything that starts with 4.11.0 should allow converged flow
+	version.PreRelease = ""
+	SupportConvergedFlow := version.Compare(*semver.New(MinimalVersionForConvergedFlow)) >= 0
```

Comment:
can't we use https://github.com/openshift/assisted-service/blob/master/internal/common/version.go#L5 ?

Reply:
We can, thanks

Reply:
OK, it's not ignoring `PreRelease`, I'll extend it